src/rluni/controller/fullrobot/controllerABC.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from abc import ABC, abstractmethod
+from collections import namedtuple
+from dataclasses import dataclass
+
+@dataclass 
+class EulerAngles:
+    x: float
+    y: float
+    z: float
+
+@dataclass 
+class EulerRates:
+    x: float
+    y: float
+    z: float
+
+@dataclass
+class MotorSpeedsRPM:
+    pitch: float
+    roll: float 
+    yaw: float
+
+# Consider how we are representing data
```

Comment:
There is duplication here could be removed. We have a telemetry dataclass that packages some of these things in a different way. If we are going to have euler angles rates and motor speeds, then control input should be those three objects as fields.

Reply:
It could add more clarity if we switch to pitch, roll, yaw instead of xyz when it comes to the robot state for a controller. 

x is roll
y is pitch
z is yaw